cdk.json

```diff
@@ -27,6 +27,9 @@
     "@aws-cdk/core:target-partitions": [
       "aws",
       "aws-us-gov"
-    ]
+    ],
```

Comment:
I don't think it makes a difference, but thought I'd ask. On line 14 is `yarn.lock` are we using `yarn` somewhere?  Also, on line 16 is `test`, we are no longer using a `test` folder anymore, so `test` is not needed is it?

Reply:
Oh yeah, everything in that block above is part of the default template from `cdk init`. I think it's for the `cdk watch` command which we don't really use (or at least I don't _think_ any of us do but I'd love to see someone try it). So are the default feature flags in the context.

Reply:
Oh okay. Yeah, it does look like it's for the `cdk watch` which can also use `hot-swap` that you mentioned previously. This will help speed up deployment, so if I get a chance I'll try to use it.